src/routes/relay/entities/schemas/relay.legacy.dto.schema.ts

```diff
@@ -1,17 +1,11 @@
-import { RelayLegacyDto } from '@/routes/relay/entities/relay.legacy.dto.entity';
-import { JSONSchemaType } from 'ajv';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import { z } from 'zod';
 
-export const RELAY_LEGACY_DTO_SCHEMA_ID =
-  'https://safe-client.safe.global/schemas/relay/relay.legacy.dto.json';
-
-export const relayLegacyDtoSchema: JSONSchemaType<RelayLegacyDto> = {
-  $id: RELAY_LEGACY_DTO_SCHEMA_ID,
-  type: 'object',
-  properties: {
-    chainId: { type: 'string' },
-    to: { type: 'string' },
-    data: { type: 'string' },
-    gasLimit: { oneOf: [{ type: 'string' }, { type: 'null', nullable: true }] },
-  },
-  required: ['chainId', 'to', 'data'],
-};
+export const RelayLegacyDtoSchema = z.object({
```

Comment:
nitpick: to move the schema closer to the entity. In this case to `src/routes/relay/entities/relay.legacy.dto.entity.ts` (I think it helps with code navigation – less jumps).

Reply:
Moved in 08933097cdc0dcb961862e1daabdd681dd331b20.